src/pkg/p2p/preheat/models/extensions.go

```diff
@@ -0,0 +1,241 @@
+package models
```

Comment:
If we do not need this file now, maybe we can delete it.

Reply:
It's need by the old code from p2p branch, I'll refactoring it later.